lib/fisk.rb

```diff
@@ -113,6 +113,29 @@ def value
       end
     end
 
+    AH = Register.new "ah", "r8", 0
+    CH = Register.new "ch", "r8", 1
+    DH = Register.new "dh", "r8", 2
+    BH = Register.new "bh", "r8", 3
```

Comment:
Do you know how we should differentiate the 8-bit high-registers, compared to the low-registers? They're both `r8`.